rpc/common/moc_common_computecommon.proto

```diff
@@ -129,3 +129,14 @@ message VirtualMachineRunCommandInstanceView {
 	string Output = 3;
 	string Error = 4;
 }
+
+enum Architecture {
```

Comment:
Please can you add a comment on where you are getting these values from?

Reply:
Values fetched from Win32_Processor class.

Documentation here:
https://docs.microsoft.com/en-us/windows/win32/cimwin32prov/win32-processor#properties